.changeset/witty-adults-sin.md

```diff
@@ -0,0 +1,5 @@
+---
+"@yamada-ui/charts": major
```

Comment:
I think you need to add “@yamada-ui/theme”.

Reply:
@zonoryo17 

Thank you for your advice.
I've fixed and updated that.